db/crud.go

```diff
@@ -1684,6 +1686,11 @@ func (db *DatabaseContext) assignSequence(ctx context.Context, docSequence uint6
 			if err != nil {
 				return unusedSequences, err
 			}
+			numSkippedSequences := docSequence - doc.Sequence
```

Comment:
```suggestion
			numReleasedSequences := docSequence - firstAllocatedSequence
```
We use the 'skipped' terminology when talking about sequences being processed by the caching DCP feed, and 'unused' or 'released' sequences when talking about sequence allocation at write time.

What we're trying to report here is the jump in _sync:seq, so we want to compare docSequence against firstAllocatedSequence and not doc.Sequence.
However, it would be even better to update nextSequenceGreaterThan to return the exact of released sequences, and report that. I think we should take that approach, instead of guessing about what the sequence allocator is doing.